Codeinwp/neve
# Table editing feedback

### Summary
Adds left and right border when a user is editing table contents: https://vertis.d.pr/NiGpOD

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Create a block table
- Try editing it, you should be able to tell that you're inside a column and which column you're in


closes #2427
